examples/src/main/java/com/mapbox/navigation/examples/core/InstructionViewActivity.kt

```diff
@@ -78,6 +83,7 @@ class InstructionViewActivity :
     private var mapboxNavigation: MapboxNavigation? = null
     private var navigationMapboxMap: NavigationMapboxMap? = null
     private lateinit var speechPlayer: NavigationSpeechPlayer
+    private lateinit var mapboxRouteAlert: MapboxRouteAlert
```

Comment:
`MapboxRouteAlert` does not seem to handle style changes internally, and it's not recreated when a new style is provided either. I didn't test this manually, but looks like the symbols would disappear when the style changes during a nav session, did you verify this scenario?

Reply:
that's a good finding. I didn't realize this before. Now it's addressed and verified. Let me know if the new approach has any problems.